CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ All notable changes to this project will be documented in this file.
 - Replace all `SystemAnimator` classes with `SystemTransitionAnimator`
 [#427](https://github.com/IBAnimatable/IBAnimatable/pull/427) by [@SD10](https://github.com/sd10)
 - `PresentationDesignable` now supports `contextFrameForPresentation` which allow you to present a controller with a custom configuration over another instead of being in fullscreen. Imitates `UIModalPresentationStyle.currentContext`
+- Remove the use of `self` where not necessary
```

Comment:
@SD10 sorry, I don't think we need to put the title as "Remove the use of `self` where not necessary" for breaking changes, because it doesn't make sense for the user. I meant we should specify the API changes. something like Change the initializer for `PortalAnimator`, `NatGeoAnimator`, `TurnAnimator`, `CardsAnimator`, `FlipAnimator`, `SlideAnimator`, `ZoomAnimator` and `DropDownAnimator`. Rename the parameter from `transitionDuration` to `duration`.